scripts/build.sh

```diff
@@ -11,14 +11,13 @@ conda list
 echo "***** Make temp directory *****"
 TEMP_DIR=$(mktemp -d)
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-cd ${DIR}/..
+# Maybe this shoudl be exported in the same way we have feestock_root???
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+FEEDSTOCK_ROOT=${SCRIPT_DIR}/..
```

Comment:
I would prefer to edit those scripts if it's not necessary since it was working fine before. Drone does not look easy to set up, maybe you should try in another PR so we can this one more readable. Drone is not required for merging this PR.

Reply:
i agree. want me to force push to delete the commits?

Reply:
Yes please and let's address drone in another PR.

Reply:
Please just reconfirm that you want **me** to force push.

Reply:
```
git reset --hard 727b924
git push origin aarch64 --force
```

I let you do it so you can keep your commits at your convenience.

Reply:
Done!